packages/aws-rfdk/lib/deadline/scripts/bash/enableCacheFilesd.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Script to install and configure cachefilesd onto a Linux host.
+# This script is presently narrowly written for use on the ECS container host
+# instance(s) that are running the Deadline RCS containers in the RenderQueue construct.
+
+set -xeu
+
+if ! test -f /sbin/cachefilesd
+then
+  # The yum install can fail if there is no route to the yum repository.
+  # Since this is an AL2 instance, that is equivalent to requiring a route to
+  # the regional S3 endpoint on port 80.
+  sudo yum install -y cachefilesd || ( echo 'ERROR -- Failed to install cachefilesd' && exit 0 )
```

Comment:
```suggestion
  sudo yum install -y cachefilesd || echo 'ERROR -- Failed to install cachefilesd' && exit 0
```
Looks like we are now creating a subshell and running `exit` from there, so the script will continue to run even if the yum install fails. This is different from what we had before. Should we get rid of the subshell?

Reply:
Drat. Right you are.